index.js

```diff
@@ -1207,6 +1207,10 @@ ArrayTome.prototype.shift = function () {
 		diff(this, 'shift');
 	}
 
+	if (ObjectTome.isObjectTome(out)) {
```

Comment:
Both tomes and untomes have their appeal, I'd lean in favor of tomes.

I guess the real issue is that any outstanding refs still need to be "fixed". They have been destroyed, but we don't have a guard that makes sure diffs don't propagate from destroyed tomes.

I think the simplest approach would be to have destroy remove `__root__` from each tome and have diff return early if no `__root__`.

We don't have to care about emitting because we remove all listeners on destroy.